dlib/external/pybind11/tools/FindPythonLibsNew.cmake

```diff
@@ -113,21 +113,18 @@ list(GET _PYTHON_VALUES 7 PYTHON_LIBRARY_SUFFIX)
 list(GET _PYTHON_VALUES 8 PYTHON_LIBDIR)
 list(GET _PYTHON_VALUES 9 PYTHON_MULTIARCH)
 
-# Ignore this test while crosscompiling otherwise it will use the host python.
-IF(NOT CMAKE_CROSSCOMPILING)
```

Comment:
Seems like we should keep this.  It was added in #1848 for a seemingly still valid reason.